src/app/new-auth/new-auth.module.ts

```diff
@@ -0,0 +1,36 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { NewAuthComponent } from './new-auth.component';
+import { RegisterComponent } from './register/register.component';
+import { LoginComponent } from './login/login.component';
+import { ChangePasswordComponent } from './change-password/change-password.component';
+import { ForgotPasswordComponent } from './forgot-password/forgot-password.component';
+import { EmailVerifiedComponent } from './email-verified/email-verified.component';
+import { Routes, RouterModule } from '@angular/router';
+import { NewAuthRoutingModule } from './new-auth.routing';
+import { InputFieldComponent } from './components/input-field/input-field.component';
+import { MatInputModule } from '@angular/material/input';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { ErrorBannerComponent } from './components/error-banner/error-banner.component';
+
+@NgModule({
+  declarations: [
+    NewAuthComponent,
+    RegisterComponent,
+    LoginComponent,
+    ChangePasswordComponent,
+    ForgotPasswordComponent,
+    EmailVerifiedComponent,
+    InputFieldComponent,
+    ErrorBannerComponent
+  ],
+  imports: [
+    CommonModule,
+    NewAuthRoutingModule,
+    RouterModule,
+    MatInputModule,
+    FormsModule,
+    ReactiveFormsModule
```

Comment:
Just checking, we need both FormsModule and ReactiveFormsModule?

Reply:
yeah, FormsModule is for ngForm and ngModel, ReactiveFormsModule is for formControl. there may be another way to do it but for now whenever i remove either, it throws errors